docs/tutorials/06_torch_runtime.ipynb

```diff
@@ -0,0 +1,1621 @@
+    "                \"loss\": 0.7771394729614258,\n",
+    "                \"forward_time\": 0.004325389862060547,\n",
+    "                \"backward_time\": 0.14366017580032348,\n",
+    "                \"epoch_time\": 2.972076654434204,\n",
+    "            },\n",
+    "            {\n",
+    "                \"epoch\": 3,\n",
+    "                \"loss\": 0.803067970275879,\n",
+    "                \"forward_time\": 0.004159808158874512,\n",
+    "                \"backward_time\": 0.14785971641540527,\n",
+    "                \"epoch_time\": 3.0526950359344482,\n",
+    "            },\n",
+    "            {\n",
+    "                \"epoch\": 4,\n",
+    "                \"loss\": 0.8006114959716797,\n",
+    "                \"forward_time\": 0.0044343829154968265,\n",
+    "                \"backward_time\": 0.14634374380111695,\n",
+    "                \"epoch_time\": 3.0279295444488525,\n",
+    "            },\n",
+    "        ],\n",
+    "        \"validation\": [],\n",
+    "    },\n",
+    "    \"execution_time\": 15.419778823852539,\n",
+    "}\n",
+    "\n",
+    "\n",
+    "torch_runtime_client._model.load_state_dict(str_to_obj(result_[\"model_state_dict\"]))\n",
+    "\n",
+    "fit_result = TorchRuntimeResult()\n",
+    "fit_result.job_id = 12345\n",
+    "fit_result.train_history = result_[\"train_history\"][\"train\"]\n",
+    "fit_result.execution_time = result_[\"execution_time\"]\n",
+    "fit_result.model_state_dict = str_to_obj(result_[\"model_state_dict\"])\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "id:  12345\n",
+      "execution time:  15.419778823852539\n",
+      "train history:  [{'epoch': 0, 'loss': 1.192161750793457, 'forward_time': 0.008829903602600098, 'backward_time': 0.15732910633087158, 'epoch_time': 3.3365063667297363}, {'epoch': 1, 'loss': 0.7761648178100586, 'forward_time': 0.004354143142700195, 'backward_time': 0.14646663665771484, 'epoch_time': 3.029676914215088}, {'epoch': 2, 'loss': 0.7771394729614258, 'forward_time': 0.004325389862060547, 'backward_time': 0.14366017580032348, 'epoch_time': 2.972076654434204}, {'epoch': 3, 'loss': 0.803067970275879, 'forward_time': 0.004159808158874512, 'backward_time': 0.14785971641540527, 'epoch_time': 3.0526950359344482}, {'epoch': 4, 'loss': 0.8006114959716797, 'forward_time': 0.0044343829154968265, 'backward_time': 0.14634374380111695, 'epoch_time': 3.0279295444488525}]\n",
+      "validation history:  None\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"id: \", fit_result.job_id)\n",
+    "print(\"execution time: \", fit_result.execution_time)\n",
+    "print(\"train history: \", fit_result.train_history)\n",
+    "print(\"validation history: \", fit_result.val_history)\n"
+   ]
```

Comment:
Same here, a lot of raw numbers, data description and/or a plot would be better.